integrations/langchain/src/databricks_langchain/genie.py

```diff
@@ -36,7 +39,12 @@ def _query_genie_as_agent(input, genie_space_id, genie_agent_name):
 
 
 @mlflow.trace(span_type="AGENT")
-def GenieAgent(genie_space_id, genie_agent_name: str = "Genie", description: str = ""):
+def GenieAgent(
+    genie_space_id,
+    genie_agent_name: str = "Genie",
+    description: str = "",
+    client: Optional["WorkspaceClient"] = None,
```

Comment:
QQ - is this the same argument name for VectorSearchRetrieverTool and UCToolkit? Actually, I guess `VectorSearchRetrieverTool` doesn't support a client yet right? https://api-docs.databricks.com/python/databricks-ai-bridge/latest/databricks_langchain.html#databricks_langchain.VectorSearchRetrieverTool

Reply:
VectorSearch is different. Does not take in a workspace client. I Have a different PR for it coming up, just need to finish tests.